pkg/token/options_test.go

```diff
@@ -149,9 +166,53 @@ func TestOptionsWithEnvVars(t *testing.T) {
 			}
 			o.AddFlags(&pflag.FlagSet{})
 			o.UpdateFromEnv()
-			if o != tc.expected {
+			if !cmp.Equal(o, tc.expected, cmp.AllowUnexported(Options{})) {
 				t.Fatalf("expected option: %+v, got %+v", tc.expected, o)
 			}
 		})
 	}
 }
+
+func TestParsePoPClaims(t *testing.T) {
```

Comment:
This can be easly converter to a TDT model to avoid duplication

```

func Test(t *testing.T) {
	testCases := []struct {
		desc	string
		
	}{
		{
			desc: "",
			
		},
	}
	for _, tC := range testCases {
		t.Run(tC.desc, func(t *testing.T) {
			
		})
	}
}
```

Reply:
Converted to this model